app/Http/Controllers/TableController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Organization;
+use Inertia\Inertia;
+use Inertia\Response;
+
+class TableController extends Controller
+{
+    public function __construct() {}
+
+    public function show(): Response
+    {
+        $organizations = Organization::with("repositories.workflowRuns.workflowJobs")->with("repositories.workflowRuns.workflowActor")->get();
```

Comment:
```suggestion
        $organizations = Organization::with(["repositories.workflowRuns.workflowJobs", "repositories.workflowRuns.workflowActor"])->get();
```
It looks better, lets think about getting data from `WorkflowJob` model and add to query some with relationship queries. (to avoid 4 time foreach)